PlannTer/Modules/SettingsModule/SettingsView.swift

```diff
@@ -64,9 +64,7 @@ struct SettingsView: View {
                     )
                 }
                 Spacer()
-                //TODO: what about action here?
-                LargeButton(title: "Save Settings", action: { })
-                Spacer()
+                
```

Comment:
this is a good choice, but wouldn't it look better if we left a button just to dismiss the view?
just a suggestion though